crates/oxc_linter/src/lib.rs

```diff
@@ -18,13 +19,16 @@ mod service;
 mod settings;
 mod utils;
 
+include!(concat!(env!("OUT_DIR"), "/environments.rs"));
```

Comment:
can we manually import this instead of including the file here. I try to avoid this kind of things in oxc to reduce dependency complexity.

Reply:
It is included explicitly in https://github.com/oxc-project/oxc/pull/2130/files#diff-207225884c5e031ffd802bb99e4fbacbd8364b1343a1cec5485bf50f29186300R12 now.